Modules/DevTools/TestFramework/TestLibraries/Any/src/Any.Codeunit.al

```diff
@@ -44,8 +44,17 @@ codeunit 130500 "Any"
     /// <param name="DecimalPlaces">Number of decimal places</param>
     /// <returns>Pseudo-random decimal value</returns>
     procedure DecimalInRange(MaxValue: Integer; DecimalPlaces: Integer): Decimal
+    var
+        x: Integer;
```

Comment:
I would rename x to RandomIntegerInRange. This is much more understandable for future code readers

Reply:
PseudoRandomInteger? It is not random it is PseudoRandom. InRange is just implementation detail, thus I would not reflect it in the name. It is shorter as well.

Reply:
@nikolakukrika 
>  It is not random it is PseudoRandom.

Just for my information; could you please explain the difference?

Reply:
If you call Library Any or Library Random and ask for randIntInRange 5 times and get 11, 25, 1, 2, 4
When this transaction ends and you call the same function again, you will get the same numbers - 11, 25, 1, 2, 4. 

With true random, you would get a completely different set of numbers.

This behavior is essential for the tests, otherwise we would not be able to reproduce failures on a local box. Pseudorandom is achieved by always setting a seed for the Any library and in a weird way in Library Random : ). I will implement clearing the seed after every test method for library any, so the tests in a codeunit and the tests individually will always get the same numbers.